rest/access_test.go

```diff
@@ -111,20 +111,20 @@ func TestStarAccess(t *testing.T) {
 	assert.NoError(t, a.Save(bernard))
 
 	// GET /db/docid - basic test for channel user has
-	response := rt.Send(RequestByUser("GET", "/db/doc1", "", "bernard"))
+	response := rt.SendUserRequest("GET", "/db/doc1", "", "bernard")
```

Comment:
I didn't switch any tests by eye and since this test uses the default collection anyway until https://github.com/couchbase/sync_gateway/pull/5990 lands, I was going to let the person who fixes the test. "/db" is correct here because the keyspace is `db._default._default`